docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,395 @@
+
+   * *(Optional)* To override the default values,
+     :ref:`create your own values.yaml <creating-your-own-values-yaml>` to
+     define the parameters of your deployment. To run the install command using
+     the ``values.yaml``, add the ``f`` parameter in you ``install`` command.
+
+     .. code-block:: text
+
+         helm install -f values.yaml my-trino-cluster trino/trino
+
+   You should see output as follows:
+
+   .. code-block:: text
+
+       NAME: my-trino-cluster
+       LAST DEPLOYED: Tue Sep 13 14:12:09 2022
+       NAMESPACE: default
+       STATUS: deployed
+       REVISION: 1
+       TEST SUITE: None
+       NOTES:
+       Get the application URL by running these commands:
+         export POD_NAME=$(kubectl get pods --namespace default -l "app=trino,release=my-trino-cluster,component=coordinator" -o jsonpath="{.items[0].metadata.name}")
+         echo "Visit http://127.0.0.1:8080 to use your application"
+         kubectl port-forward $POD_NAME 8080:8080
+
+   This output depends on your ``values.yaml`` configuration and cluster name.
+   For example, the port ``8080`` is set by the ``.service.port`` in the
+   ``values.yaml``.
+
+#. Run the following command to check that all pods, deployments, and services
+   are running properly.
+
+   .. code-block:: text
+
+      kubectl get all
+
+   You should expect to see output that shows running pods, deployments, and
+   replica sets. A good indicator that everything is running properly is to see
+   all pods are returning a ready status in the  ``READY`` column.
+
+   .. code-block:: text
+
+       NAME                                               READY   STATUS    RESTARTS   AGE
+       pod/my-trino-cluster-coordinator-bfb74c98d-rnrxd   1/1     Running   0          161m
+       pod/my-trino-cluster-worker-76f6bf54d6-hvl8n       1/1     Running   0          161m
+       pod/my-trino-cluster-worker-76f6bf54d6-tcqgb       1/1     Running   0          161m
+
+       NAME                       TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)    AGE
+       service/my-trino-cluster   ClusterIP   10.96.25.35   <none>        8080/TCP   161m
+
+       NAME                                           READY   UP-TO-DATE   AVAILABLE   AGE
+       deployment.apps/my-trino-cluster-coordinator   1/1     1            1           161m
+       deployment.apps/my-trino-cluster-worker        2/2     2            2           161m
+
+       NAME                                                     DESIRED   CURRENT   READY   AGE
+       replicaset.apps/my-trino-cluster-coordinator-bfb74c98d   1         1         1       161m
+       replicaset.apps/my-trino-cluster-worker-76f6bf54d6       2         2         2       161m
+
+   * The top three lines are pods. These include the actual Trino containers.
+
+   * The next line is the service. This is a mechanism by which Kubernetes makes
+     internal services like the one running on the coordinator accessible
```

Comment:
like the discovery service running on the coordinator

Reply:
Also .. I assume the chart handles that automatically so that the workers can contact the coordinator

Reply:
If you talk purely from a chart's perspective then you can remove all explanations since it can be considered implementation detail of the chart. The `service` in this case indeed gets created by the chart - nothing the user needs to do. I think this paragraph and explanation of output above is explaining k8s concepts more than anything specific to Trino or the charts.

Reply:
@mosabua and @hashhar, perhaps we should remove this section and just link to these resources:

[kubectl get reference](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#get)
[example](https://kubernetes.io/docs/reference/kubectl/_print/#docker-ps)

WDYT?